test/server/rest/base.spec.js

```diff
@@ -13,7 +13,8 @@ process.env.NODE_ENV = process.env.NODE_ENV || "test-rest-spec";
 
 var sinon = require("sinon");
 var app = require("../../../server");
-var PORT = process.env.TEST_REST_PORT || 3020;
+var testPort = 3020;
```

Comment:
GLOBAL: If you have a top level variable that isn't going to change after initialization, use `CAPS_CASE`